### PR TITLE
dev-erlang/*: version bumps, EAPI8 bumps

### PR DESCRIPTION
**Pull request by mm1ke**
Some easy version and `EAPI8` bumps.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by Flowdalic** [comment]
Thanks, looks good to me. Will give it a test run.

Note that luerl is a potential last-rite candidate.

File: dev-erlang/eredis/eredis-1.2.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+
+DEPEND=">=dev-lang/erlang-17.1"
+RDEPEND="${DEPEND}"
+
+DOCS=( AUTHORS CHANGELOG.md README.md )
+
+# Needs running redis instance at 127.0.0.1:6379.
+RESTRICT=test
```
Comment (ConiKost) [inline dev-erlang/eredis/eredis-1.2.0-r1.ebuild:23]: According to ```skel.ebuild```, it comes after ```IUSE.```

File: dev-erlang/meck/meck-0.9.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+
+DEPEND=">=dev-lang/erlang-17.1"
+RDEPEND="${DEPEND}"
+
+DOCS=( CHANGELOG.md NOTICE README.md )
+
+# Tests need rebar3. The build somewhat works with rebar2.
+RESTRICT="test"
```
Comment (ConiKost) [inline dev-erlang/meck/meck-0.9.2.ebuild:22]: According to ```skel.ebuild```, it comes after ```IUSE.```

File: dev-erlang/proper/proper-1.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+KEYWORDS="~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+
+DEPEND=">=dev-lang/erlang-17.1:="
+RDEPEND="${DEPEND}"
+
+# tests broken with current erlang 23.x
+RESTRICT="test"
+
+src_configure() {
+	./configure
```
Comment (ConiKost) [inline dev-erlang/proper/proper-1.4.ebuild:24]: Handcrafted script?

**Review by ConiKost — CHANGES_REQUESTED** [review]

File: dev-erlang/proper/proper-1.4.ebuild (hunk 3 above)
Comment (Flowdalic) [reply dev-erlang/proper/proper-1.4.ebuild:24]: Thanks for pointing this out. The whole `src_configure` definition can be dropped. Going to do so in a follow up commit.

**Comment by mm1ke** [comment]
thx @ConiKost and @Flowdalic for the push :)

**Pull request closed without merging**